README.yaml

```diff
@@ -55,63 +55,78 @@ description: |-
 
 quickstart: |-
 
-  Here's how you can quickly demo the `bastion`. We assume you have `~/.ssh/authorized_keys` properly configured and your SSH key (e.g. `~/.ssh/id_rsa`) added to your SSH agent.
+  Here's how you can quickly demo the `bastion`. This is not a Quick Start guide for production / actual usage.
+
+  ### Assumptions
+
+  - We assume you have `~/.ssh/authorized_keys` properly configured 
+  - Your SSH key (e.g. `~/.ssh/id_rsa`) added to your SSH agent.
 
 
+  As a non-escalated user:
   ```bash
-  $ docker run -it -p 1234:22 \
-       -e MFA_PROVIDER=google-authenticator \
-       -v ~/.ssh/authorized_keys:/root/.ssh/authorized_keys
-       cloudposse/bastion
+  docker run -it -p 1234:22 \
+      -e MFA_PROVIDER=google-authenticator \
+      -v ~/.ssh/authorized_keys:/root/.ssh/authorized_keys \
+      cloudposse/bastion
   ```
 
-  Now, in another terminal you should be able to run:
+  Now, let's attempt to connect to the Bastion:
   ```bash
-  $ ssh root@localhost -p 1234
+  ssh root@localhost -p 1234
   ```
 
-  The first time you connect, you'll be asked to setup your MFA device. Subsequently, each time you connect, you'll be prompted to enter your MFA token.
+  The first time you connect, you'll be asked to setup your MFA device ([GIF](docs/demo.gif)). Subsequently, each time you connect, you'll be prompted to enter your MFA token.
 
-# How to use this project
 usage: |-
+
+  ### MFA Providers
+
+  This documentation only recognises Google Authenticator and Duo.
+
+  - Google Authenticator
+    - Google Authenticator is a free, open-source, and easy to setup MFA.
+    - It's less secure than Duo because tokens are stored on the server under each user account.
+  - Duo
+    - Duo is an enterprise paid-for service. It has excellent push-notification support so that MFA requests can be approved or declined without opening an app.
```

Comment:
```suggestion
    - Duo is a paid enterprise service. It has excellent mobile push-notification support so that MFA requests can be approved or declined without opening an authenticator app.
```